packages/graphql-lowcode/src/generate/roots/roots.ts

```diff
@@ -34,5 +37,41 @@ export function getRoots(types: TypesObject[], rootNames: { type: string, name?:
   const mutationRoot = roots.find(root => root.name === mutation.name)
   const subscriptionRoot = roots.find(root => root.name === subscription.name)
 
-  return[queryRoot, mutationRoot, subscriptionRoot]
+  return [queryRoot, mutationRoot, subscriptionRoot]
+}
+
+export function getQueryNames(introspection: IntrospectionQuery, entityName: string): (string | undefined)[] {
+  const rootNames = getRootNames(introspection)
+  const [queryRoot, mutationRoot, subscriptionRoot] = getRoots(introspection.types, rootNames)
+
+  const listTypeQuery = queryRoot?.fields.find(field => isListType(field)) ?? queryRoot?.fields[0] //TODO zmenit
+  const detailTypeQuery = queryRoot?.fields.find(field => isObjectType(field) ?? queryRoot?.fields[0])
+
+  //TODO update, insert etc...
+
+
+  return [listTypeQuery?.name, detailTypeQuery?.name]
```

Comment:
this is a anti pattern - you should return an object - something like `{ list, detail }`